requirements.txt

```diff
@@ -13,9 +13,9 @@ pyserial>=3,<=3.4
 Jinja2>=2.7.3,<=2.10
 intelhex>=1.3,<=2.2.1
 intervaltree>=2,<3
-mbed-ls>=1.5.1,<1.7
+mbed-ls>=1.5.1,<=1.7
```

Comment:
`<=1.7` means it will pick up 1.7.0, not 1.7.1, correct?

Is this intentional since the `0.2` release updated mbed-ls to 1.7.1?

Reply:
Oh that's a good point. I'll need to correct this.

Reply:
Oh man yeah, I tried this locally and it did indeed only install 1.7.0! Thanks again!